src/classes/box.cpp

```diff
@@ -386,3 +386,18 @@ Vec3<double> Box::scaleFactors(double requestedVolume, Vec3<bool> scalableAxes)
 
     return {scalableAxes.x ? ratio : 1.0, scalableAxes.y ? ratio : 1.0, scalableAxes.z ? ratio : 1.0};
 }
+
+toml::basic_value<toml::discard_comments, std::map, std::vector> Box::serialize()
+{
+    toml::basic_value<toml::discard_comments, std::map, std::vector> box;
+
+    toml::array boxLengths = {a_, b_, c_};
+    toml::array boxAngles = {alpha_, beta_, gamma_};
+    toml::array boxNonPeriodic = {!periodic_.x, !periodic_.y, !periodic_.z};
+
+    box["lengths"] = boxLengths;
+    box["angles"] = boxAngles;
+    box["nonPeriodic"] = boxNonPeriodic;
```

Comment:
```suggestion
    box["lengths"] = toml::array{a_, b_, c_};
    box["angles"] = toml::array{alpha_, beta_, gamma_};
    box["nonPeriodic"] = toml::array{!periodic_.x, !periodic_.y, !periodic_.z};
```

Shorten the code with fewer variables.